src/LibVLCSharp.WPF/ForegroundWindow.cs

```diff
@@ -69,7 +69,7 @@ void Background_Unloaded(object sender, RoutedEventArgs e)
 
         void Background_Loaded(object sender, RoutedEventArgs e)
         {
-            if (_wndhost != null)
+            if (_wndhost != null && IsVisible)
```

Comment:
Do we want `IsEnabled` here instead?

Reply:
Not sure what you mean here? if it's hidden by the tab control, I'm not sure IsEnabled will be false